Not reading ice server credentials & ontrack issue with WASM

I am opening a peer connection via the WASM example and i'm using my turn server in the config with all the info (url, username, credential) but in the browser i get error:
`JavaScript error: Failed to construct 'RTCPeerConnection': Both username and credential are required when the URL scheme is "turn" or "turns".`

I build the config with the following code:

	config := webrtc.Configuration{
		ICEServers: []webrtc.ICEServer{
			{
				URLs:       []string{"turn:myturnserver:443?transport=tcp"},
				Username:   "myusername",
				Credential: "mycredential",
			},
		},
	}
	

Also if i try to use OnTrack i get error again:
```
./main.go:65:4: pc.OnTrack undefined (type *webrtc.PeerConnection has no field or method OnTrack)
./main.go:65:22: undefined: webrtc.TrackRemote
```

--- a/peerconnection_js.go
+++ b/peerconnection_js.go
@@ -568,11 +568,10 @@ func iceServersToValue(iceServers []ICEServer) js.Value {
 
 func iceServerToValue(server ICEServer) js.Value {
 	return js.ValueOf(map[string]interface{}{
-		"urls":     stringsToValue(server.URLs), // required
-		"username": stringToValueOrUndefined(server.Username),
-		// Note: credential and credentialType are not currently supported.
-		// "credential":     interfaceToValueOrUndefined(server.Credential),
-		// "credentialType": stringEnumToValueOrUndefined(server.CredentialType.String()),
+		"urls":           stringsToValue(server.URLs), // required
+		"username":       stringToValueOrUndefined(server.Username),
+		"credential":     interfaceToValueOrUndefined(server.Credential),
+		"credentialType": stringEnumToValueOrUndefined(server.CredentialType.String()),
 	})
 }
 

